app/utils.py

```
@@ -0,0 +1,51 @@
+        str(input_path),
+        "-vn",
+        "-acodec",
+        "pcm_s16le",
+        "-ac",
+        str(channels),
+        "-ar",
+        str(sample_rate_hz),
+        str(output_path),
+    ]
+
+    process = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)